src/wallet/wallet2.h

```diff
@@ -1214,6 +1214,7 @@ namespace tools
     uint64_t get_blockchain_height_by_date(uint16_t year, uint8_t month, uint8_t day);    // 1<=month<=12, 1<=day<=31
 
     bool is_synced() const;
+    bool is_close_to_synced() const;
```

Comment:
Rather than a new method that is almost identical to the existing one, let's just add an optional "grace_blocks" parameter into the existing is synced, something like:
```C++
bool is_synced(uint64_t grace_blocks = 0);
```
and then the caller can use `is_synced(2)` or whatever grace blocks they want (and existing `is_synced()` will do the same thing as it does now).